code/controllers/CMSMain.php

```diff
@@ -964,6 +964,12 @@ public function save($data, $form) {
 			$record = $this->getNewItem($id, false);
 		}
 
+		// Check publishing permissions
+		$doPublish = !empty($data['publish']);
+		if ($record && $doPublish && !$record->canPublish()) {
```

Comment:
Would you mind removing this permission check? It's probably not going to cause any problem for any one, but this is going to be our last SS3 release and we would rather only do bug fixes and avoid introducing new behavior that doesn't directly fix a bug.

Reply:
Sure, I've removed the perms check.